### ISSUE TITLE
Automatic stop monitoring an artist

### ISSUE THREAD

**Issue by Ruke805** [digitalec/deemon#33]
I had this situation:

I have a Music folder that is monitored by Deemon, in a moment I just stopped following some artists, but Deemon is keeping monitoring and downloading from them. 

In my case: 
I removed my Hans Zimmer folder, I just have an album from him and I don't want more, but Deemon continues downloading new releases, and I don't want to. I was wondering if have some way to auto-remove an artist who no longer has a folder.

**Comment by digitalec** [comment]
As of deemon 1.0, monitored artists are stored and managed in the database rather than read in each time (0.4.x and earlier). You can remove an artist from monitoring using `deemon monitor --remove Hans Zimmer`.

I'm going to leave this open and mark as a feature request though. Perhaps in one of the next updates I can add in the ability to actively monitor a directory.

**Comment by Ruke805** [comment]
> I'm going to leave this open and mark as a feature request though. Perhaps in one of the next updates I can add in the ability to actively monitor a directory.

Yeah, could be as an additional flag, It doesn't need to check some removed artists each time. It would be a nice feature

**Comment by digitalec** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Automatic stop monitoring an artist](https://trello.com/c/aBQW16Ax/4-automatic-stop-monitoring-an-artist)
